### ISSUE TITLE
Exclude external storage from sync

### ISSUE THREAD

**Issue by marvelloard** [owncloud/core#20180]
Hi,

it would be nice to be able to exclude external storages (local/webdav/whatever) from any sync clients, globally in a way that those folders only appear in the webinterface, but not in the clients at all.

Let me give you an example. Some software (LibreOffice, tons of little helper tools,..) is being stored on the server. Now I want to be able to access it from some computer. Of course I can integrate it to ownCloud. But then, my sync client wants it to sync too, and I'd have to deselect the sync manually. The solution with not syncing folders >500MB doesn't quite work here since there are folders inside smaller than 500MB and computers end up having parts of this software archive.

Now, the problem becomes bigger when trying to give other users access to this software folder. They end up with parts of this archive on their HDD.

In short, an option to exclude those ext. folders from the sync client so it won't see it at all would be fantastic.

**Comment by mmattel** [comment]
Ref: #7909 Disable synchronisation for certain directories

**Comment by Xenopathic** [comment]
@mmattel Agreed, duplicate of #7909.

@marvelloard If you want, feel free to voice your support on that issue :smile:

**Comment by marvelloard** [comment]
Sorry for the duplicate, didn't realise there is an issue already!
